vars/uploadMinSnapshotsToS3.groovy

```diff
@@ -38,7 +50,7 @@ void call(Map args = [:]) {
         action(argsMap)
     }
 
-    echo('Start copying files')
+    echo("Start copying files: ver-${version} arch-${architecture} plat-${platform} build-${id} dist-${distribution} ext-${extension}")
```

Comment:
Can you use complete words here if at all required? Or was this used for debugging?

Reply:
This is just to make the line shorter.
Tho if needed I can expand it to be longer and complete.
Thanks.

Reply:
Resolved now.
Thanks.